.github/workflows/python-app.yml

```diff
@@ -0,0 +1,32 @@
+name: Python application
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
I think we should use the same Python version as the project to avoid any possible incompatibilities.

Reply:
I agree python-version: 3.7.2 would be ideal

Reply:
sounds good boys!  nice catch.